app/js/main.js

```diff
@@ -267,15 +269,15 @@ function startup() {
   btnFramePrevious.addEventListener("click", function() {
     if (curSelectedFrame > 1) {
         _displayFrame(curSelectedFrame - 1);
-    } else if (winMode === "capture") {
+    } else if (winMode === "capture" && totalFrames) {
```

Comment:
@le717 This change prevents the user from accessing playback mode when no frames have been captured via the previous frame button.

It appears I've already independently made this change on `master`.